Makefile.am

```diff
@@ -8,6 +8,10 @@ oci_systemd_hook_LDADD += $(SELINUX_LIBS)
 oci_systemd_hook_CFLAGS += $(LIBMOUNT_CFLAGS)
 oci_systemd_hook_LDADD += $(LIBMOUNT_LIBS)
 
+oci_systemd_hook_json_CONFIG = oci-systemd_hook-json
```

Comment:
should thist be "{dash}json" rather than "{dot}json"?

Reply:
The DATA file below has the .json file.  It looks like thie object is not used in the generated Makefile.  I am not even sure if it necessary.

Reply:
Looks like I can just remove the line.

Reply:
Removed.